src/main/java/andersen/lesson/hashset/Main.java

```diff
@@ -1,12 +1,16 @@
-package hashset;
+package andersen.lesson.hashset;
 
 public class Main {
     public static void main(String[] args) {
```

Comment:
You should avoid having multiple main methods in your program because it would be ambiguous which one to run